Formula/snowcli.tmpl.rb

```diff
@@ -9,12 +9,11 @@ class Snowcli < Formula
 
     def install
       ENV["CARGO_NET_GIT_FETCH_WITH_CLI"] = "true"
-      #without_pip=false because of https://github.com/Homebrew/brew/pull/15792
-      venv = virtualenv_create(libexec, "python3", system_site_packages: false, without_pip: false)
+      venv = virtualenv_create(libexec, "python3", system_site_packages: false)
       venv.instance_variable_get(:@formula).system venv.instance_variable_get(:@venv_root)/"bin/python",
-        "-m", "pip", "install", "pip==22.3.1"
-      venv.instance_variable_get(:@formula).system venv.instance_variable_get(:@venv_root)/"bin/pip",
-        "install", "snowflake-cli-labs=={{ version }}"
+        "-m", "ensurepip", "--upgrade"
+      venv.instance_variable_get(:@formula).system venv.instance_variable_get(:@venv_root)/"bin/python",
+        "-m", "pip", "install", "snowflake-cli-labs==2.0.0"
```

Comment:
If we don't use `version`, let's remove it from `update.py`

Reply:
We should - `update.py` replaces `{{ version }}` from `snowcli.tmpl.rb` into actual version in `snowcli.rb`

```suggestion
        "-m", "pip", "install", "snowflake-cli-labs=={{ version }}"
```

Reply:
Changed